crates/abq_cli/src/abq_config.rs

```diff
@@ -0,0 +1,121 @@
+use anyhow::anyhow;
+use std::{fs, path::PathBuf};
+
+use abq_hosted::AccessToken;
+use etcetera::{app_strategy, AppStrategy, AppStrategyArgs};
+use serde_derive::{Deserialize, Serialize};
+
+#[derive(Deserialize, Serialize, Debug, PartialEq, Eq)]
+pub struct AbqConfig {
+    pub rwx_access_token: AccessToken,
+}
+
+pub enum AbqConfigFileMode {
+    // Conventional unix location (e.g ~/.config/abq/config.toml)
+    Conventional,
+    // Non-standard location (e.g. ~/my/custom/path/config.toml)
+    Override(String),
+    // Ignore any located config file (For instance, if the host machine happens to have a config file)
+    Ignore,
+}
+
+pub fn abq_config_filepath(mode: AbqConfigFileMode) -> Option<PathBuf> {
+    match mode {
+        AbqConfigFileMode::Ignore => None,
+        AbqConfigFileMode::Override(path) => Some(PathBuf::from(path)),
+        AbqConfigFileMode::Conventional => {
+            let strategy = app_strategy::Unix::new(AppStrategyArgs {
+                top_level_domain: "org".to_string(),
+                author: "rwx".to_string(),
+                app_name: "abq".to_string(),
+            });
```

Comment:
can we use `choose_app_strategy` (https://docs.rs/etcetera/latest/etcetera/index.html#appstrategy) rather than tying to unix, if it doesn't require a material change of the API? If it does, let's stick with this.

Reply:
I ended up going this route because of [this](https://docs.rs/etcetera/latest/etcetera/index.html#appstrategy):

> Apple strategy would place these in ~/Library/Preferences/org.acmecorp.FrobnicatorPlus.

Let me know if you're seeing something I'm not 😅

Reply:
Ah, thanks! Okay, let's keep as-is.